package.json

```diff
@@ -302,7 +293,7 @@
     "build-static-viz:watch": "webpack --config webpack.static-viz.config.js --watch",
     "build-static-viz:watch-wait": "yarn wait:cljs; yarn build-static-viz:watch",
     "build-stats": "yarn && webpack --json > stats.json",
-    "build-storybook": "yarn build:cljs && NODE_OPTIONS=--max-old-space-size=8196 build-storybook",
```

Comment:
scripts are defined in the `storybook` folder package.json, so I we explicitly say it should be started there